lib/bob/job/build_elixir.ex

```diff
@@ -12,43 +12,22 @@ defmodule Bob.Job.BuildElixir do
     case ref do
       "v0" <> _ -> ["17.3"]
       "v1.0.0-" <> _ -> ["17.3"]
-      "v1.0.0" -> ["17.3"]
-      "v1.0.1" -> ["17.3"]
-      "v1.0.2" -> ["17.3"]
-      "v1.0.3" -> ["17.3"]
+      "v1.0." <> patch when patch in ~w(0 1 2 3) -> ["17.3"]
       "v1.0.4" -> ["17.5"]
       "v1.0.5" -> ["17.5"]
       "v1.0" <> _ -> ["17.3", "18.3"]
```

Comment:
Also could it be that the `v1.0x` mapping is wrong here?
The above lines tell that:

versions from `1.0.0-rc-x` to `1.0.3` work with erlang `17.3`
versions from `1.0.4` to `1.0.5` work with erlang `17.5`
versions from `1.0.6` up to latest `1.0.x` work with `17.3` and `18.3`

except that looking at https://github.com/elixir-lang/elixir/tags?after=v1.1.0 there doesn't seem to be any version after `1.0.5` and that is explicitly mapping to `17.5`

is this line irrelevant?
```elixir
      "v1.0" <> _ -> ["17.3", "18.3"]
```

or it is for mapping the `v1.0` tag?
in this case, why is it `["17.3", "18.3"]` if none of the 1.0.x versions map to `18.3` ?

Reply:
This seems to be a mistake yes, it should either be `["17.5", "18.3"]` or `["17.5"]`. Probably safest to go with the latter.